chapters/core/guides/index.rst

```diff
@@ -38,4 +38,5 @@ Further information on any specific method can be obtained in the :doc:`../api_r
    ../../evaluation
    ../../incremental_indexing
    ../../stress/index
+   ../../io/index
```

Comment:
Should we flatten this to just 
`chapters/input_output.rst`

Reply:
done